### DISCUSSION TITLE
Coupon codes

### DISCUSSION THREAD

**Discussion by dawidurbanski** [wpappointments/wpappointments#98] [Ideas]
Coupon codes

**Comment by deaf-potato** [comment]
Personally, I would like to have a choice in which database to keep them in. I would like to be able to use the codes that are entered through woocommerce. You could then, for example, sell Gift Vouchers that reduce the price of booking a service.

**Comment by wppoland** [comment]
Thanks for the input, and sorry for the long silence here. The direction we settled on is WooCommerce as an optional integration (see discussion #76): with it enabled, a booking creates a WooCommerce order, so coupon and gift voucher codes are handled by your store checkout rather than duplicated inside the plugin. A separate coupon store inside the plugin is not planned for v1.0. Current focus is shipping v1.0 of the free core plugin.
